data/resources/ui/device/row.ui

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+
+  <object class="GtkSizeGroup">
+    <property name="mode">horizontal</property>
+    <widgets>
+      <widget name="mode_box"/>
+      <widget name="host_path_entry"/>
+    </widgets>
+  </object>
+
+  <object class="GtkSizeGroup">
+    <property name="mode">horizontal</property>
+    <widgets>
+      <widget name="container_path_entry"/>
+    </widgets>
+  </object>
```

Comment:
> One other thing that I couldn't figure out was how to make the container path and host path text inputs the same width in the DeviceRow. Seems that the width of the host path text input is wider because of the first row that contains the switches.

You're correct. The mode box makes the host path entry too wide. Removing all groups will fix it.

```suggestion
```


With the other suggestions below, the end result will look like

![grafik](https://user-images.githubusercontent.com/3630213/193918606-ae9c8caa-5a32-444e-b13d-cae63b325754.png)